EDU-10252 - Update vtex-store-icons-iconpack.md (#739)
vtexdocs/dev-portal-content: docs/vtex-io/Store-Framework-Apps/advanced-components/vtex-store-icons-iconpack.md

    mutedClassName: PropTypes.string,
  }
  export default IconSearch
 ```

## SVG Prefixes

> The following prefixes help to understand what a specific icon means.

Author: ```suggestion
The following prefixes help in understanding the meaning of a specific icon.
```
